lib/upfluence/endpoint/api_endpoint.rb

```diff
@@ -38,6 +39,18 @@ def respond_with(resource, *args)
           halt [status, result] || 404
         end
       end
+
+      def json_params
+        begin
+          JSON.parse(request.body.read)
```

Comment:
The think is, rbutils should make `active_support` dependent all packages who using it ?